executor/wasm-host/src/host.rs

```diff
@@ -1238,7 +1256,9 @@ pub fn casper_transfer<S: GlobalStateReader + 'static, E: Executor>(
         }
         Err(error) => {
             error!(?error, "Error while reading from storage; aborting");
-            panic!("Error while reading from storage; aborting")
+            // TODO: Is this panic-worthy?
```

Comment:
I'm fine with an internal host error with the assumption that contract runtime can catch internal error, and then decide to panic the node, or handle it any other way that's safe for the network.